packages/desktop/src/components/channel-message.js

```diff
@@ -325,6 +325,7 @@ const ChannelMessage = React.memo(function ChannelMessage_({
         <AppMessage
           isHovering={isHovering}
           message={message}
+          onClickInteractiveElement={onClickInteractiveElement}
```

Comment:
this feels odd, but not sure how best to do it without moving the whole `onClickInteractiveElement` method out of the `ChannelMessage` component. suggestions welcome!

Reply:
Would be prettier to compose the RichText one level up perhaps, but I think this is fine for now.

```js
// e.g.
<AppMessage
  message={
    <RichText
      blocks={message.content}
      onClickInteractiveElement={onClickInteractiveElement}
    />
  }
/>